README.md

```diff
@@ -1,56 +1,39 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# To-Do List
+![ScreenShot](/images/Capture.JPG)
+> In this project, a simple to-do list is builded.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- CSS
+- JavaScript
+- HTML
+- npm
+- Webpack
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://fsher07.github.io/To-Do-List/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+-To run the project locally, clone the project using the commands and link below:
 
-### Run tests
-
-### Deployment
+-git clone https://github.com/Fsher07/plain-js-with-objects.git
 
+### Setup
 
+-To run the project locally, clone the project using the commands and link below:
 
+-git clone https://github.com/Fsher07/plain-js-with-objects.git
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.